cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -149,6 +168,28 @@ func doSystemdMountImpl(what, where string, opts *systemdMountOptions) error {
 	if opts.Private {
 		options = append(options, "private")
 	}
+	if opts.Overlayfs {
+		if len(opts.LowerDirs) <= 0 || len(opts.UpperDir) <= 0 || len(opts.WorkDir) <= 0 {
+			return fmt.Errorf("cannot mount %q at %q: missing arguments for overlayfs mount. lowerdir, upperdir, workdir are needed.", what, where)
+		}
+
+		escaper := strings.NewReplacer(",", `\,`, " ", `\ `)
```

Comment:
We need to escape `\` as well. This is a valid path character on Linux.

Reply:
I forbid the use of `\` in all overlayfs mount options and the what and where arguments. In a subsequent PR I will try to remove all escaping and expand the list of forbidden characters to also include ` `(space),`,` and `:`. I will keep allowing `:` for the lowerdir option though.